Gaijinizer patch/Map042.txt

```diff
@@ -87,113 +91,128 @@ again?#MTLed
-About "male"en(v[44] \>= 40)#MTLed
+That "Male"en(v[44] \>= 40)#Ron
 > END STRING
 
 > BEGIN STRING
 『バーゲスト』 en(v[44] \>= 60) 
 > CONTEXT: Map042/events/5/pages/1/6/Choice/2
-"Barghest"en(v[44] \>= 60)#MTLed
+Barghestsen(v[44] \>= 60)#Ron
 > END STRING
 
 > BEGIN STRING
 さんぽ en(v[44] \>= 80) 
 > CONTEXT: Map042/events/5/pages/1/6/Choice/3
-Sanpoen(v[44] \>= 80)#MTLed
+Go For A Walken(v[44] \>= 80)#Ron
 > END STRING
 
 > BEGIN STRING
 あ、あのあと　わたし　を
 
  ヘカーテ　さま　よんだ。
 > CONTEXT: Map042/events/5/pages/1/11/Dialogue/enemy2
-Oh, after that I. Hecate I am.#MTLed
+A-, after that... I... called out to...
+Hecate-sama.#Ron
 > END STRING
 
 > BEGIN STRING
 わたし　消される　おもって
 
  カクゴ　して　出てきた。
 > CONTEXT: Map042/events/5/pages/1/15/Dialogue/enemy2
-I will be erased. It has come out.#MTLed
+I thought... I would be... erased...
+
+I had... prepared... myself... for that...#Ron
 > END STRING
 
 > BEGIN STRING
 でも　ヘカーテ　さま　すごい　ニコニコ
 
  わたし　しゃべる　ようにした
 > CONTEXT: Map042/events/5/pages/1/19/Dialogue/enemy2
-But Hecate-sama is amazing * smile *
-I was supposed to talk#MTLed
+But... Hecate-sama... had this... amazing...
+
+smile... I tried my best... to speak...#Ron
 > END STRING
 
 > BEGIN STRING
 そして　言った
 
 『にどと　もどる　ダメ』って……
 > CONTEXT: Map042/events/5/pages/1/24/Dialogue/enemy3
-And said. What's "Don't Back"#MTLed
+And... she said... 
+
+"You can... never come... back home."#Ron
 > END STRING
 
 > BEGIN STRING
 ………要するに、捨てられちゃったのか……。
 > CONTEXT: Map042/events/5/pages/1/28/Dialogue/kazuya
-... In short, has it been abandoned ....#MTLed
+... In other words, she abandoned you...#Ron
 > END STRING
 
 > BEGIN STRING
 それにしても、片言だなんて
 
 もっと滑らかに喋れるようにしてくれても良かっただろうに…
 > CONTEXT: Map042/events/5/pages/1/30/Dialogue/kazuya
-Even so, it's a word. It would have been nice to
-be able to drown more smoothly ...#MTLed
+Not only that but you've got that broken speech...
+It would've been nice if she'd given you the
+ability to speak normally...#Ron
 > END STRING
 
 > BEGIN STRING
 ウゥ……　もともと　わたし　くち
 
  しゃべる　コウゾウ　できて　ない。
 > CONTEXT: Map042/events/5/pages/1/34/Dialogue/enemy2
-Wow ...... originally I. I can't talk.#MTLed
+Ugh... I was not... originally... constructed...
+with the... ability... to speak...#Ron
 > END STRING
 
 > BEGIN STRING
 あまり　はやく　しゃべる　とっ
 
  たくさん　ベロ　からまる……
 > CONTEXT: Map042/events/5/pages/1/38/Dialogue/enemy2
-I talk too fast. Lots of Vero ...#MTLed
+When I... speak... too quickly... my tongues...
+get twisted up...#Ron
 > END STRING
 
 > BEGIN STRING
 ああ………
 
 あの大量の舌が、むしろ邪魔してるのか…。
 > CONTEXT: Map042/events/5/pages/1/42/Dialogue/kazuya
-Oh………. Is that a lot of tongue rather disturbing
-....#MTLed
+Oh...
+
+I guess having that many tongues is actually quite
+a nuisance for you...#Ron
 > END STRING
 
 > BEGIN STRING
 まぁ、シェルター襲撃用に『特製で』創ったって
 
  ヘカーテも言ってたしなぁ……
 > CONTEXT: Map042/events/5/pages/1/46/Dialogue/kazuya
-Well, I made it "specially made" for shelter
-attack. He also said Hecate ...#MTLed
+Well, she did say she made you "specially" to
+attack the shelter...#Ron
```

Comment:
"Specially" here feels a bit stiff in English. From what I'm reading from the translations, it gives the impression that Empuse was Special/Custom made for the attack, which fits with what we know.

Let's try something like this:
>_**Well, she did say she "hand make" you to
attack the shelter...**_

(Custom make is also viable here, but I feel this is the best trade off.)